frontend/router/index.tsx

```diff
@@ -264,10 +262,8 @@ const routes = (
             </Route>
             <Route path=":id" component={PolicyPage} />
           </Route>
-          <Route
-            path="profile"
-            component={UserSettingsPage as RouteComponent}
-          />
+          <Redirect from="profile" to="account" /> {/* deprecated URL */}
+          <Route path="account" component={AccountPage as RouteComponent} />
```

Comment:
I see this is legacy stuff, but it looks like a `RouteComponent` is just any class or function component, so I don't think this casting is necessary

Reply:
100, ty!